scrapy/crawler.py

```diff
@@ -264,6 +270,28 @@ def get_spider_middleware(self, cls: type[_T]) -> _T | None:
             )
         return self._get_component(cls, self.engine.scraper.spidermw.middlewares)
 
+    async def crawl_async(self, *args: Any, **kwargs: Any) -> None:
+        if self.crawling:
+            raise RuntimeError("Crawling already taking place")
```

Comment:
Could this be a good chance to increase coverage? (I did not check if the coverage data is accurate or we do have tests for these but they are executed in a way that is not reflected in the coverage data)